src/Http/Controllers/UserController.php

```diff
@@ -69,6 +70,13 @@ public function store(UserRequest $request, $id): JsonResponse
             }
         }
 
+        if (
+            ! isset($data['locale'])
```

Comment:
I wonder if we even need to check if it's set or not here? Or would it suffice to simply verify if it's in the `availableLanguageCodes()`?

Reply:
It seems good, I was a bit iffy about it, as I don't know exactly what Laravel does when a validation rule allows null. (Thought it might omit the key entirely, if it is omitted from the request)

Reply:
I take that back, it seems my assumptions were correct.

<img width="727" alt="Screenshot 2020-11-26 at 03 28 59" src="https://user-images.githubusercontent.com/2234034/100301042-961d4580-2f97-11eb-964e-1294cd2c827f.png">

I'll re-add the check

Reply:
Thanks for checking!